.env.test

```diff
@@ -5,15 +5,18 @@ OTEL_RESOURCE_ATTRIBUTES=deployment.environment=test
 
 # Translate Service
 TRANSLATE_SERVICE_PORT=8080
-TRANSLATE_SERVICE_HOST=localhost
+TRANSLATE_SERVICE_HOST=0.0.0.0
 TRANSLATE_SERVICE_DB=mysql
 
 # MySQL
-TRANSLATE_DB_MYSQL_HOST=localhost
+TRANSLATE_DB_MYSQL_HOST=0.0.0.0
 TRANSLATE_DB_MYSQL_PORT=3306
 TRANSLATE_DB_MYSQL_USER=root
 TRANSLATE_DB_MYSQL_PASSWORD=
 TRANSLATE_DB_MYSQL_DATABASE=translate
 
 # BadgerDB
 TRANSLATE_DB_BADGERDB_PATH=/tmp/badger
+
+# Google Translate API
+TRANSLATE_TRANSLATE_SERVICES_GOOGLE_TRANSLATE_API_KEY=
```

Comment:
seems a bit too verbose. Word TRANSLATE gets repeated 3 times.

Reply:
first translate is a prefix, not debatble

second translate is the one which could be debatable

third one is a google_translate, not debatble

Reply:
how about GOOGLE_TRANSLATE_API_KEY ?

Reply:
the `TRANSLATE` prefix must be present
and we will have multiple translate_services, so some intermediate is needed, and `translate_services` should suffice
```yaml
translate_services:
  google_translate:
    api_key: ...
    #others service specific keys
  another_service:
   ...
```

this is `examples/translate.yaml` and env variables must be the same layout, e.g. prefix + path to key, and in this case `TRANSLATE_TRANSLATE_SERVICES_GOOGLE_TRANSLATE_API_KEY`

Reply:
We have to simplify it. I suggest a general purpose `TRANSLATE_OTHER` settings for external APIs - TRANSLATE_OTHER_GOOGLE_TRANSLATE_API_KEY.